R/get_metadata.R

```diff
@@ -0,0 +1,11 @@
+#' @title Get metadata
+#'
+#' @param dataset_name character name of dataset
```

Comment:
Is there an overarching place people can find the list of `dataset_name` options? Should reference it here.